### PR TITLE
Migrate to use Xenial in Travis

### PR DESCRIPTION
**Pull request by cherifGsoul**
This pull request was created with [Landscaper](https://github.com/bitovi/landscaper).
The following code mods were used to create this PR:

1. **https://gist.github.com/cherifGsoul/def29ecbfa9ff53b859d466c7929870c**

Please review this PR carefully as Landscaper does not guarantee any code mod's quality.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=lock
```
Comment (chasenlehara) [inline .npmrc]: Is there a reason this is `lock` and not `false`?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js: node
-script: npm test
```
Comment (chasenlehara) [inline .travis.yml]: Should this have been removed?

File: .npmrc (hunk 1 above)
Comment (cherifGsoul) [reply .npmrc]: Mistake!

File: .travis.yml (hunk 2 above)
Comment (cherifGsoul) [reply .travis.yml]: The build fails when this line is present.

**Pull request merged**